app.js

```diff
@@ -0,0 +1,28 @@
+const http = require('http')
+const express = require('express')
+const routes = require("./routes");
+const cors = require('cors');
+
```

Comment:
```suggestion
const http = require('http')
const express = require('express')
const cors = require('cors');
const routes = require("./routes");
```
npm으로 설치하는 라이브러리를 먼저 require 하시고, 우리가 만든 모듈을 require하시는 것이 통상적으로 가독성 높은 코드 순서입니다 :)